e2e/tests/meditor.spec.ts

```diff
@@ -0,0 +1,80 @@
+import { test, expect, type Page } from "@playwright/test";
+import { clientUrl, gotoAndLogin } from "../utils.ts";
+
+const noLicenseSelector = async (page: Page) => {
+  await page.getByRole("tab", { name: "General Badge" }).click();
+  await expect(page.getByText("required information")).toBeVisible();
+};
+const noContributorSelector = async (page: Page) => {
+  await expect(
+    page.getByRole("tab", { name: "Dandiset contributors" }),
+  ).toBeVisible();
+};
+const invalidLicenseSelector = async (page: Page) => {
+  await page.getByRole("tab", { name: "General Badge" }).click();
+  await expect(page.getByText('must be equal to one of the')).toBeVisible();
+};
+const titleTooLongSelector = async (page: Page) => {
+  await page.getByRole("tab", { name: "General Badge" }).click();
+  await expect(
+    page.getByText("must NOT be longer than 150 characters"),
+  ).toBeVisible();
+};
+
+const dandisetsToTest = [
+  "000003",
+  "000018",
+  "000024",
+  // "000071",
+  "000106",
+  // "000107",
```

Comment:
I wonder if there's value in testing at least one of `000071` or `000107`. Since there's no other dandisets being tested that have two simultaneous validation errors. What do you think?

Reply:
Theses are commented out due to https://github.com/dandi/dandi-archive/issues/2281. Although, looking at this again, Dandiset 000024 also tests for `noContributorSelector`, so I'm a bit confused about that. I'll take a closer look.

Reply:
We can actually re-enable to tests for 000107. The actual assertion that fails is the icon indicating that the metadata is invalid. Both 000024 and 000107 have additional validation errors on the main tab that result in the correct icon being displayed. 000071 is special in that the ONLY validation error is on the contributors tab.